src/Handlers/VersionExtension.cs

```diff
@@ -58,6 +59,13 @@ public static IApplicationBuilder UseVersion(this IApplicationBuilder builder)
                 {
                     // return the version info
                     context.Response.ContentType = "text/plain";
+
+                    if (App.Config.BurstHeader)
+                    {
+                        string serviceName = string.IsNullOrWhiteSpace(App.Config.BurstService) ? VersionExtension.Name : App.Config.BurstService;
```

Comment:
Instead of doing the check for the BurstService name in 3 separate locations, let's do the check once when the Config is set.  Then the conditional is only executed once at start-up and the rest of the code may just retrieve the value.

Reply:
I was trying to go this route originally but wasn't able to get the reflection information from the commandline file. I'll see if there's a way to get the reflection to complete first.